report/21_Process-Developer-Interaction.md

```diff
@@ -1 +1,5 @@
-How do you interact as developers?
+The developers interact mainly through a weekly meeting right after the lecture, through the use of a Discord Server, and through the use of the GitHub Repository on which the project is hosted. After the lockdown caused be Covid-19, the weekly meeting started being held over Discord as well. 
+Discord has been used for direct communication, both in text-channels for quick questions and requests, and through voice channels for meetings or collaborative work.
```

Comment:
Maybe add some examples of channels, like the ohdear, github and review channel which made communication clearer?

Reply:
ohdear og github channels aren't communication though. They are monitoring tools